docs/rules/eventmap-params.md

```diff
@@ -70,6 +76,54 @@ Template.foo.events({
 
 ```
 
+#### Destructuring
+
+You can optionally allow destructuring the parameters.  
+You can set `disallowDestructuring` to `"event"`, `"templateInstance"`, or `"both"`, to force no destructuring on the event parameter, template-instance parameter, or both respectively.
```

Comment:
This is a nitpick, but the plugin already contains a rule which uses `prevent` as the verb to not allow something.
Can you change every occurrence of `disallow` to `prevent`?